### PR TITLE
Algebraic Team Feedback

### PR DESCRIPTION
**Pull request by vlin02**

### CODE REVIEW

File: exploration/5. score_criteria_reevaluation/2. m2_sigmoid_alg_fitting.ipynb [hunk 1]
```diff
@@ -0,0 +1,310 @@
...
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": []
+  }
+ ],
```
Comment (Cozieee) [inline exploration/5. score_criteria_reevaluation/2. m2_sigmoid_alg_fitting.ipynb:291]: I'd recommend also visualizing them overlapped:
```
beatmap_id = 104229
f, ax = plt.subplots(figsize = (14, 12))

x = get_x_and_y(beatmap_id)[0]
y = get_x_and_y(beatmap_id)[1]

popt = fit_log(beatmap_id)
plt.plot(x, logiFunc(x, *popt), label = 'Fitted logistic function')

popt = fit_genLog(beatmap_id)
plt.plot(x, genLogFunc(x, *popt), label = 'Fitted logistic function')

plt.plot(x,y,label = 'original')
plt.title('Fitting generalized logistic function for beatmap %d' %(beatmap_id))
plt.legend()
```

File: exploration/5. score_criteria_reevaluation/2. m2_sigmoid_alg_fitting.ipynb [hunk 2]
```diff
@@ -0,0 +1,310 @@
...
+   "source": [
+    "def plot_fit_log(popt, beatmap_id):\n",
+    "    \n",
+    "    x = get_x_and_y(beatmap_id)[0]\n",
+    "    y = get_x_and_y(beatmap_id)[1]\n",
+    "    \n",
+    "    f, ax = plt.subplots(figsize = (14, 12))\n",
+    "    plt.title('Fitting logistic function for beatmap %d' %(beatmap_id))\n",
+    "    plt.plot(x,y,label = 'original')\n",
+    "    plt.plot(x, logifunc(x, *popt), 'r-',label = 'Fitted logistic function')\n",
```
Comment (Cozieee) [inline exploration/5. score_criteria_reevaluation/2. m2_sigmoid_alg_fitting.ipynb:241]: should be logiFunc

File: exploration/5. score_criteria_reevaluation/2. m2_sigmoid_alg_fitting.ipynb [hunk 3]
```diff
@@ -0,0 +1,310 @@
...
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "def fit_log(beatmap_id):\n",
+    "    \n",
+    "    x = get_x_and_y(beatmap_id)[0]\n",
+    "    y = get_x_and_y(beatmap_id)[1]\n",
+    "    \n",
+    "    popt, pcov = curve_fit(logifunc, x, y, maxfev = 1000)\n",
```
Comment (Cozieee) [inline exploration/5. score_criteria_reevaluation/2. m2_sigmoid_alg_fitting.ipynb:203]: should be logiFunc

File: exploration/5. score_criteria_reevaluation/2. m2_sigmoid_alg_fitting.ipynb [hunk 4]
```diff
@@ -0,0 +1,310 @@
...
+   "source": [
+    "# You can run any of the following\n",
+    "# popt = fit_alg(104229)\n",
+    "# plot_fit_alg(popt, 104229)\n",
+    "\n",
+    "# popt = fit_alg(104229)\n",
+    "# plot_fit_alg(popt, 104229)\n",
+    "\n",
+    "# popt = fit_alg(104229)\n",
+    "# plot_fit_alg(popt, 104229)"
```
Comment (Cozieee) [inline exploration/5. score_criteria_reevaluation/2. m2_sigmoid_alg_fitting.ipynb:281]: These 3 are all the same.

**Comment by vlin02** [comment]
You also have several notebooks that are conflicting to merge, even though you shouldn't have done any work in them. See if you can fix that.

**Comment by marcoscialanga** [comment]
> You also have several notebooks that are conflicting to merge, even though you shouldn't have done any work in them. See if you can fix that.

This is weird, we didn't modify all of those. what should we do?

**Pull request merged**